docs/src/basics/job-workers.md

```diff
@@ -4,33 +4,40 @@ A job worker is a component capable of performing a particular step in a workflo
 
 ## What is a Job?
 
-A job is a _work item_ in a workflow. For example:
+A job is a *work item* in a workflow. For example:
 
 * Processing a payment
 * Generating a PDF document
 * Updating customer data in a backend system
 
 A job has the following properties:
 
-* **Type**: Describes the work item and is defined in each task in the workflow model. The type is referenced by workers to request the jobs they are able to perform.
-* **Payload**: The business data the worker should work on in the form of a JSON document/object.
-* **Headers**: Additional metadata defined in the workflow, e.g. describing the structure of the payload.
+* **Type**: Describes the work item and is defined in each task in the workflow. The type is referenced by workers to request the jobs they are able to perform.
+* **Variables**: The contextual/business data of the workflow instance that is required by the worker to do its work.
+* **Custom Headers**: Additional static metadata defined in the workflow. Mostly used to configure a worker which is used for more than one workflow step.
 
-## Requesting Jobs From The Broker
+## Requesting Jobs from the Broker
```

Comment:
I think it was intended that all words begin with an upper case letter :sweat_smile:

Reply:
On other places, we don't start "filler" words with an upper case. I think it's ok ;)

Reply:
ok